### PR TITLE
fix --tag-prefix option

### PR DESCRIPTION
**Pull request by daniel-faber**

### CODE REVIEW

File: util.js [hunk 1]
```diff
@@ -43,7 +44,7 @@ function urlTemplate (opts) {
   var hostMirrorUrl = getHostMirrorUrl(opts)
```
Comment (ralphtheninja) [inline util.js]: There's a `v{version}` on line 43 as well

**Comment by ralphtheninja** [comment]
Some stuff needs tweaking still

* `getOpts()` in `test/download-test.js` needs to have `'tag-prefix'` property default to 'v'.
* tests in `test/util-test.js` needs updating (the ones testing url template)
* optionally, add a `|| 'v'` to `util.js`: `tag_prefix: opts['tag-prefix'] || 'v'`

File: util.js (hunk 1 above)
Comment (daniel-faber) [reply util.js]: I think it is correct not to change this line.

In my project my dependency `@serialport/bindings` fails to install.

It calls `prebuild-install --tag-prefix @serialport/bindings@`.

The correct download url is https://github.com/node-serialport/node-serialport/releases/download/@serialport/bindings@2.0.2/bindings-v2.0.2-node-v57-linux-x64.tar.gz

As you see the `v` in `v2.0.2-node...` must not be replaced by `@serialport/bindings@`.

**Comment by daniel-faber** [comment]
I completely missed the tests, sorry! Will look into it now.

**Comment by daniel-faber** [comment]
All tests green again.

File: util.js (hunk 1 above)
Comment (ralphtheninja) [reply util.js]: > As you see, `v` in `v2.0.2-node...` must not be replaced by `@serialport/bindings@`.

:+1:

**Comment by ralphtheninja** [comment]
5.2.0

**Comment by daniel-faber** [comment]
Version 5.2.0 works as expected, thank you! 👍

**Pull request merged**